obj.js

```diff
@@ -431,10 +433,7 @@ function getExistingIDForLocalObject(obj) {
  * @returns {string} ID
  */
 function getNewIDForLocalObject(obj) {
-  let id;
-  do {
-    id = generateObjID();
-  } while (gLocalIDsToObjects.has(id))
+  let id = generateNewObjID();
```

Comment:
As discussed, this should stay a string ID.

Reply:
This was fixed in PR#1.